.openshift/templates/deployment.yml

```diff
@@ -94,31 +94,30 @@ objects:
         volumes:
         - name: cert-volume
           secret:
-            secretName: site-int-cert
+            secretName: ${APP_NAME}-int-cert
         - name: config-volume
           configMap:
-            name: site-config
+            name: ${APP_NAME}-config
     test: false
     triggers:
     - type: ConfigChange
     - imageChangeParams:
         automatic: true
         containerNames:
-        - site
+        - ${APP_NAME}
         from:
           kind: ImageStreamTag
-          name: site:latest
-          namespace: uncontained--prototype
+          name: ${APP_NAME}:latest
       type: ImageChange
 - apiVersion: v1
   kind: Service
   metadata:
     annotations:
-      service.alpha.openshift.io/serving-cert-secret-name: site-int-cert
+      service.alpha.openshift.io/serving-cert-secret-name: ${APP_NAME}-int-cert
```

Comment:
As you may already know, browsers don't trust this certificate for our domain:


>uncontained.io uses an invalid security certificate.
>
>The certificate is only valid for the following names: *.b9ad.pro-us-east-1.openshiftapps.com, b9ad.pro-us-east-1.openshiftapps.com

Should I file a separate issue for getting a new cert?

Reply:
@rdebeasi this doesn't currently affect us, as we are not using https. This will come into play when we move to the Managed Platform at some point. For now we can leave it.